### ISSUE TITLE
[Question] Column series with (conditional)icon + text + hideOversized 

### ISSUE THREAD

**Issue by FrankMerema** [amcharts/amcharts4#1389]
HI,

I'm trying to add some icons to the labelBullets added to the series and find out that I can use the html field for this.  So far I have:
```
labelBullet.label.adapter.add('html', (value, target) => {
  let icon = '';
  if ((target.dataItem.dataContext as Task).solution) {
    icon = (target.dataItem.dataContext as Task).completed ? '<i class="ok"></i>' : '<i class="progress"></i>';
  }

  return `${icon}<span>${(target.dataItem.dataContext as Task).description}</span>`
});
```
The issue which I'm facing now is that the `hideOversized = true` is not working anymore and the chart is polluted with labels.

Is there any way I can fix it?

**Comment by martynasma** [comment]
Looks like `hideOversized` is ignored for HTML labels. I'm not sure if this is an oversight or was made on purpose.

Would you be able to produce some sort of demo so that I can have something to work with while testing further?

**Comment by martynasma** [comment]
Nevermind my last request. We've already fixed that in dev. Will be in the next version.

**Comment by martynasma** [comment]
Fixed in [4.5.3](https://github.com/amcharts/amcharts4/releases/tag/4.5.3).

## [4.5.3] - 2019-06-18

### Added
- `"up"`, `"down"`, `"left"` and `"right"` options added to `Tooltip.pointerOrientation`. Allows forcing constant position/orientation of the Tooltip. Note, the tooltip might go out of div/screen boundaries when using these new orientations.
- New property added to `SliceGrouper`: `groupProperties`. It's an object comprising by key/value pairs to apply to a "group slice", e.g. `grouper.groupProperties.fill = am4core.color("red")`.
- It was impossible to reuse single pattern for multiple `Sprite` elements, as disposing one was resulting in pattern to be disposed as well.

### Fixed
- `hideOversized` was being ignored for HTML-based labels.
- `PieChart` series' `legendSettings` were shared across all series; it was impossible to set different `valueText`.
- `CategoryAxis` with HTML labels could in result excessive white space at the bottom of the chart.
- In some cases patterns set as `Sprite`'s fills using `fill` adapter were not displayed.
- When calculating totals on a Series with all zero values, `percent` was not being calculated. It will now have a zero.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
